homework/english/fyodorov/lecture3/vocabulary.md

```diff
@@ -1 +1,7 @@
-
+1. I will buy new iPhone because my old one has little memory. 
+2. He will write clean code next week.
+3. Will I create constructor and test soon?
+4. You will find new job as soon as will finish java courses.
+5. I will set up all parameters default.
+6. Will I merge the current branch?
+7. Will this method be explicit?
```

Comment:
I thought, the task was to write a story, but not a separated sentenses. But I'm not sure)))